src/api/spec/controllers/webui/project_controller_spec.rb

```diff
@@ -208,17 +209,86 @@
 
   describe 'GET #new' do
     before do
-      login(user_moi)
+      login user_moi
       get :new, name: 'ProjectName'
     end
 
     it { expect(assigns(:project)).to be_a(Project) }
     it { expect(assigns(:project).name).to eq('ProjectName') }
   end
 
+  describe 'GET #show' do
+    before do
+      Project.any_instance.stubs(:number_of_build_problems).returns(0) # To not ask backend for build status
```

Comment:
It does. Though if someone writes a test that touches number_of_build_problems he might have a hard time debugging this. Maybe we can use a dedicated cassette for certain backend calls (for all tests)? According to [stackoverflow](http://stackoverflow.com/questions/16533980/using-specific-vcr-cassette-based-on-request) this is possible.

That way we would at least keep testing parts of number_of_build_problems.